core/src/main/java/com/example/AutomatedReviewTest.java

```diff
@@ -0,0 +1,36 @@
+package com.example;
+
+import java.util.*;
+
+// The goal of this code is to find out if code analysis will automatically add
+// commentary to a pull request when it finds problems with the code.
+
+public abstract class AutomatedReviewTest extends Object {
+
+  public boolean hello(Optional unusedRaw)
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make this line start at column 5. ([squid:IndentationCheck](https://sonarcloud.io/organizations/coderanch-corral/rules#rule_key=squid%3AIndentationCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.coderanch%3Ablackjack&pullRequest=9&issues=AWY_zChlvOM6zLh9PlcT&open=AWY_zChlvOM6zLh9PlcT)

[ISSUE_KEY]: AWY_zChlvOM6zLh9PlcT